training/model_management/components/import_model/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: import_model
 display_name: Import model
 description: Import a model into a workspace or a registry
-version: 0.0.6
+version: 0.0.7
```

Comment:
Please check registry version & update accordingly. I think we have 0.0.4 in prod?